Cargo.toml

```diff
@@ -27,6 +27,8 @@ once_cell = "1.9.0"
 serde = "1.0.136"
 serde_json = "1.0.78"
 serde_yaml = "0.8.23"
+paste = "1.0.5"
+nomad-types = {git = "https://github.com/nomad-xyz/rust", rev = "d4d02aa"}
```

Comment:
Needed to manually point commit hashes between config and rust repos since work done in tandem. Will merge this, merge rust repo PR, then bump versions to point at latest `main`

Reply:
you can use branch specifiers here instead of commit specifiers

Reply:
Yeah so I need to use commit hashes to pin the versions to each other between the repos. I tried using branch specifiers twice and git type version conflicts :(

Reply:
This will all go away once both PRs are merged and we point each o